src/query/atomWithQuery.ts

```diff
@@ -1,16 +1,10 @@
-import { QueryClient, QueryObserver } from '@tanstack/query-core'
-import type {
-  QueryKey,
-  QueryObserverOptions,
-  QueryObserverResult,
-} from '@tanstack/query-core'
+import type { QueryKey, QueryObserverOptions } from '@tanstack/query-core'
+import { atomsWithTanstackQuery } from 'jotai-tanstack-query'
```

Comment:
Can't the naming in jotai tanstack query be like the current naming in jotai? `atomWithQuery` instead of `atomWithTanstackQuery`?

Reply:
Hm, I generally prefer shorter names. So, I'm not super happy with the current long name either.
At the same time, I think having a unique name is good for searchability.
`atoms` instead of `atom` is because we return two atoms.
Any ideas?

Reply:
Yeah, we can do that.
You mean we do naming like this for all others `jotai-urql`, `jotai-apollo`, `jotai-relay`, right?

Let's have @EduardoLopes a comment.